android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -650,31 +583,12 @@ private void setLeafAnimation(
             float leafMargin,
             boolean isTopLeaf) {
         if (leafMargin > 0 && leafAlignView != null) {
-            int margin = (int) dpToPx(this, leafMargin);
+            int margin = dpToPx(this, leafMargin);
             Animation animation =
                     new Animation() {
                         @Override
-                        protected void applyTransformation(
-                                float interpolatedTime, Transformation t) {
-                            if (leafAlignView != null) {
-                                ViewGroup.MarginLayoutParams layoutParams =
-                                        (ViewGroup.MarginLayoutParams)
-                                                leafAlignView.getLayoutParams();
-                                if (isTopLeaf) {
-                                    layoutParams.bottomMargin =
-                                            margin
-                                                    - (int)
-                                                            ((margin - layoutParams.bottomMargin)
-                                                                    * interpolatedTime);
-                                } else {
-                                    layoutParams.topMargin =
-                                            margin
-                                                    - (int)
-                                                            ((margin - layoutParams.topMargin)
-                                                                    * interpolatedTime);
-                                }
-                                leafAlignView.setLayoutParams(layoutParams);
-                            }
```

Comment:
I've extracted this logic into a separate method as the shorter line length was making it unreadable.